app/views/services/index.njk

```diff
@@ -9,6 +9,10 @@
   <script src="/public/js/components/notification-banner.js"></script>
 {% endblock %}
 
+{% block beforeContent %}
```

Comment:
Is this supposed to have anything here or is it intended ?

Reply:
It's intended, the extension mechanism in Nunjucks means that this template will by default use the `beforeContent` inherited from the main layout. As this page implements its own unique behaviour (the one page without any kind of header) it overrides this